src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -1,22 +1,14 @@
 package org.broadinstitute.consent.http.service;
 
-import java.util.ArrayList;
-import java.util.Collections;
+import java.util.*;
```

Comment:
nit: Avoid star imports here too

Reply:
It looks like this is still an issue in this file.

Reply:
Missed this one somehow, should be good now.